docs/source/changelog.rst

```diff
@@ -10,6 +10,10 @@ For more detailed information, see `GitHub <https://github.com/jupyter/notebook>
 -----
 
 - Properly handle reaping of zombie subprocesses.
+- Moved the cell toolbar selector into the view menu ![screencast](https://cloud.githubusercontent.com/assets/335567/10711889/59665a5a-7a3e-11e5-970f-86b89592880c.gif).
```

Comment:
@jdfreder Should this be `rst` or is there a cool converter that I don't know about? :question: 

Otherwise,
```
- Moved the cell toolbar selector into the view menu `[screencast] <https://cloud.githubusercontent.com/assets/335567/10711889/59665a5a-7a3e-11e5-970f-86b89592880c.gif>`__.
```